### PR TITLE
Include perseo-core.properties in WAR file

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@jcalderin

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,6 +86,13 @@
                 </includes>
                 <filtering>false</filtering>
             </resource>
+            <resource>
```
Comment (fgalan) [inline pom.xml]: Is this PR a fix for https://github.com/telefonicaid/perseo-core/issues/19?

Maybe should it be mentioned in the CHANGES_NEXT_RELEASE file?

Comment (crbrox) [inline pom.xml]: I didn't think it was relevant enough, but fixed in 47ead19

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
 Add /version.jsp path for checking version
 Decrease log level for rules
+Fix perseo-core.properties for reference missing from WAR file
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Is this PR a fix for #19? In that case, I'd suggest:

```
"Fix perseo-core.properties for reference missing from WAR file (#19)"
```

Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed 2dd9996

**Comment by fgalan** [comment]
LGTM

**Pull request merged**